### PR TITLE
Adding docker support

### PR DESCRIPTION
**Pull request by aliabbasjaffri**
## Type of Pull Request

- [ ] Bug fix
- [x] Enhancement

Related Issue #s or links (if any): 

## Description of Changes

Added support for Docker. You can find the details on this issue: https://github.com/developertools-tech/developertools.tech/issues/42

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +12,28 @@ If you have any questions about contributing after reading the guide, please cre
 
 I'm also looking for co-maintainers, if you are interested please contact me via the contact form on [dlford.io](https://www.dlford.io).
 
+## Getting Started
```
Comment (dlford) [inline README.md:15]: Please don't change the README file.

**Review by dlford — CHANGES_REQUESTED** [review]
@aliabbasjaffri Maybe I missed something but I thought the dockerizing was for development, looks like we are doing production builds here?

I don't see need for production docker builds, since this is a fully open source PWA, anyone can use it for free online or offline via the website.

**Comment by aliabbasjaffri** [comment]
Indeed! Reverting the changes to have a single click development setup using `docker-compose up`

**Comment by aliabbasjaffri** [comment]
@dlford the changes are integrated. let me know if there's anything else.

**Pull request closed without merging**